app/components/views/PrivacyPage/Privacy/PrivacyContent.jsx

```diff
@@ -60,6 +66,22 @@ const PrivacyContent = ({
       TODO
       create a shared component and use it on logs page and here.
     */}
+    <SendFromUnmixedAccountModal
+      show={showingSendUnmixModal}
+      onSubmit={onToggleSendFromUnmixed}
+      onCancelModal={() => showModal(false)}
+    />
+    <Checkbox
+      label={
+        <T
+          id="Privacy.checkbox"
```

Comment:
I think this is the only string ID I have seen starting with a capital letter. We can give it a more descriptive name too, in case we add another checkbox later.

```suggestion
          id="privacy.sendFromUnmixedCheckbox"
```